src/server/db/client.ts

```diff
@@ -16,4 +16,6 @@ const globalForPrisma = globalThis as unknown as {
 
 const prisma = globalForPrisma.prisma ?? prismaClientSingleton();
 
+if (process.env.NODE_ENV !== "production") globalForPrisma.prisma = prisma;
```

Comment:
Without this conditional, a new PrismaClient instance is created every time we refresh the page in development mode.